.eslintrc.js

```diff
@@ -1,32 +1,33 @@
 module.exports = {
+  root: true,
   env: {
     browser: true,
-    es2021: true,
-    jest: true,
+    node: true,
   },
-  extends: [
-    'eslint:recommended',
-    'plugin:react/recommended',
-    'plugin:prettier/recommended',
-    'airbnb',
-  ],
+  parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaFeatures: {
       jsx: true,
     },
-    ecmaVersion: 'latest',
-    sourceType: 'module',
   },
-  plugins: ['react'],
+  plugins: ['@typescript-eslint', 'import', 'jest', 'react'],
+  extends: [
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:jest/recommended',
+    'prettier',
+  ],
```

Comment:
Shouldn't the `jest` entries in `plugins` and `extends` be added for tests only?